tests/etc/auto-down.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+ gcloud container clusters delete warren-ds-test --region us-west4
```

Comment:
```suggestion
 gcloud container clusters delete NAME --region us-west4
```
Need a better name for the cluser? 🤷